src/routers/__tests__/StackRouter-test.js

```diff
@@ -602,15 +602,15 @@ describe('StackRouter', () => {
       {
         type: NavigationActions.SET_PARAMS,
         params: { name: 'foobar' },
-        key: 'Init',
+        key: 'Init-id-0-17',
       },
       state,
     );
     expect(state2 && state2.index).toEqual(0);
     /* $FlowFixMe */
     expect(state2 && state2.routes[0].routes[0].routes).toEqual([
       {
-        key: 'Init',
+        key: 'Init-id-0-17',
```

Comment:
This looks pretty tedious to maintain.. because one test may disrupt the key from other tests